### ISSUE TITLE
Attached configuration

### ISSUE THREAD

**Issue by sdast9** [ipc-sim/IPC#21]
This is really amazing work- thank you! I'm wondering if there is a way to maintain an attachment between 2 objects throughout the simulation. For example, in your initialize attached configuration example on the wiki, the objects fall together, but then separate after colliding with the ground. Is there a way for them to maintain the attachment (i.e. minimal distance between nodes) throughout the simulation?

**Comment by liminchen** [comment]
Great question! The simulator currently does not directly support it. You could think about implementing extra equality constraints in C++ to the system such that the relative positions of certain pairs of nodes are fixed.

**Comment by sdast9** [comment]
Agreed. I will have to learn C++ beyond "hello world" and study your code closely to get there. Just wanted to confirm that before taking on the challenge. Thanks again for this outstanding contribution!

**Comment by liminchen** [comment]
Sure! I really appreciate your interest and acknowledgment!
